src/NamespaceUsages.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types = 1);
+
+namespace Spaze\PHPStan\Rules\Disallowed;
+
+use PhpParser\Node;
+use PhpParser\Node\Name\FullyQualified;
+use PHPStan\Analyser\Scope;
+use PHPStan\Broker\ClassNotFoundException;
+use PHPStan\Rules\Rule;
+
+/**
+ * Reports on usages of classes in general.
```

Comment:
Remove the comment, it's not correct anyway. Maybe even remove the `@package` line below as well.